source/usage.rst

```diff
@@ -64,6 +64,24 @@ First, setup Homebrew [#homebrew]_ and afterwards install the RTRlib package:
 
 .. [#homebrew]  Homebrew -- http://brew.sh
 
+
+Gentoo
+------
+
+The kind guys from the Frrouting project maintain a gentoo overlay [#overlay]_ that contains an ebuild for rtrlib.
+First you have to setup layman [#layman]_, then you can install rtrlib with the following commands:
```

Comment:
for consistency (and brevity) skip `you` like above in the macOS + Homebrew instructions.